Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -53,6 +53,25 @@ void HybridPICModel::ReadParameters ()
     pp_hybrid.query("Jx_external_grid_function(x,y,z,t)", m_Jx_ext_grid_function);
     pp_hybrid.query("Jy_external_grid_function(x,y,z,t)", m_Jy_ext_grid_function);
     pp_hybrid.query("Jz_external_grid_function(x,y,z,t)", m_Jz_ext_grid_function);
+
+    pp_hybrid.query("J_external_init_style", m_J_ext_grid_s);
+
+    if (m_J_ext_grid_s == "read_from_file"){
+            const std::string read_j_fields_from_path="./";
+            pp_hybrid.query("read_j_fields_from_path", m_external_j_fields_path);
```

Comment:
I would suggest to make this:
```suggestion
            pp_hybrid.get("read_j_fields_from_path", m_external_j_fields_path);
```
that way an exception will be raised if the file path is not provided.